docs/content/docs/patterns/observability/tracing.md

```diff
@@ -36,9 +36,10 @@ The sections below outline how to make required updates depending on whether you
 If you already have OSM running, `tracing` values must be updated in the OSM ConfigMap using:
 
 ```console
-osm mesh upgrade --enable-tracing --tracing-address <tracing server hostname> --tracing-port <tracing server port> --tracing-endpoint <tracing server endpoint>
```

Comment:
Investigating why this command was changed to `kubectl patch`

Reply:
I think I suggested we change this just because it seemed easier to add the note that changes won't persist than explain how to use `osm mesh upgrade` and ensure the controller/injector images won't needlessly update and restart the control plane pods.